### DISCUSSION TITLE
Example of usage with useNavigation

### DISCUSSION THREAD

**Discussion by apdrsn** [gpbl/react-day-picker#1494] [Support] [answered]
Hi :)
Hope to find some help here 👍 

I am trying to make use of `useNavigation` to get `goToMonth` function when a user input a date to a text input, then go to that date in the calendar. 

I've created a barebones code sandbox here: https://codesandbox.io/s/react-day-picker-8-forked-th7pb7.
I don't think I get what I need to do to get it to work? I have tried to wrap the markup with `<NavigationProvider>` - no luck.

So what I wanna do is fx in this example: https://react-day-picker.js.org/guides/input-fields#example-range-selection
when user types a date into from date input field - I want the calendar to show that month.

**Comment by apdrsn** [comment, marked as answer]
Found the solution here for any others facing the same:
https://react-day-picker.js.org/basics/navigation#controlling-the-current-month

**Reply by gpbl** [reply]
Thanks @apdrsn for the feedback! We are updating the docs in #1501 so it should be more clear how to use these hooks...
